engine/lib/widgets.php

```diff
@@ -110,14 +122,22 @@ function elgg_is_widget_type($handler) {
  *
  * The widget types are \stdClass objects.
  *
- * @param string $context The widget context or empty string for current context
- * @param bool   $exact   Only return widgets registered for this context (false)
+ * @param array|string $context An associative array of options or the widget context
+ * @param bool         $exact   Only return widgets registered for this context (false)
  *
  * @return array
```

Comment:
This would now be `Elgg\WidgetDefinition[]` or would it be a mix of stdClass objects, too?